### PR TITLE
Add info on how to avoid iframe issues

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

**Comment by MarcinJuraszek** [comment]
LGTM

**Comment by KnightSky** [comment]
Looks good.

File: docs/docs_source/hostpage.rst [hunk 1]
```diff
@@ -90,6 +90,22 @@ sample host page:
     :emphasize-lines: 9, 11-29
 
 
+Working around browser iframe behavior
+--------------------------------------
+
+Some browsers exhibit strange behavior with iframes when using bookmarks or the forward/back browser buttons. In some
+cases, this will cause the Office Online iframe to be loaded twice in a single navigation. This in turn can cause
```
Comment (chrisbro-MSFT) [inline docs/docs_source/hostpage.rst]: Static frame use can also keep the frame from being recreated at all, causing the app to load with the previous session's state.

File: docs/docs_source/hostpage.rst [hunk 2]
```diff
@@ -90,6 +90,22 @@ sample host page:
...
+'file locked' errors for users. In order to work around this behavior, hosts should dynamically create the Office
```
Comment (chrisbro-MSFT) [inline docs/docs_source/hostpage.rst]: or access token expired, or invalid canary

File: docs/docs_source/hostpage.rst [hunk 3]
```diff
@@ -90,6 +90,22 @@ sample host page:
...
+Online iframe using JavaScript. This technique is illustrated in the sample host page:
```
Comment (chrisbro-MSFT) [inline docs/docs_source/hostpage.rst]: then dynamically submit it.

**Pull request merged**